### PR TITLE
Alexd02/results page

### PR DESCRIPTION
**Pull request by adundarov**

### CODE REVIEW

File: frontend/my-app/src/Pages/LandingPage.tsx [hunk 1]
```diff
@@ -167,15 +218,40 @@ const LandingPage: React.FC = () => {
...
+              id="name"
+              label="Email Address"
+              type="email"
+              fullWidth
+              variant="standard"
+            />
+          </DialogContent>
+          <DialogActions>
+            <Button onClick={handleClose}>Cancel</Button>
+            <Button onClick={handleSubmit}>Send Request</Button>
```
Comment (jspaniac) [inline frontend/my-app/src/Pages/LandingPage.tsx:245]: Seems like there are different style decisions here. For example, here we write:
`<Button onClick={handleClose}>Cancel</Button>`
one one line, but on @222 it's been broken into multiple lines. We might want to keep things as consistent as possible

File: frontend/my-app/src/Pages/LoginPage.tsx [hunk 2]
```diff
@@ -41,25 +50,95 @@ const LoginPage: React.FC = () => {
...
+                  }}
+                />
+                <TextField
+                  margin="normal"
+                  required
+                  fullWidth
+                  name="password"
+                  label="Password"
+                  type="password"
+                  id="password"
```
Comment (jspaniac) [inline frontend/my-app/src/Pages/LoginPage.tsx:112]: Honestly not sure - do we need all of these tags here?

File: frontend/my-app/src/Pages/Results.tsx [hunk 3]
```diff
@@ -26,16 +28,154 @@ const Results: React.FC = () => {
...
+                  mx: 2,
+                  p: 2,
+                  borderRadius: '10px',
+                  backgroundColor: 'success.main'
+                }}>
+                <Typography
+                  variant="h6"
+                  component="h2"
+                  textAlign="center">
+                  Average Travel Time: {result.travelTimes.reduce((a, b) => a + b, 0) / result.travelTimes.length}
```
Comment (jspaniac) [inline frontend/my-app/src/Pages/Results.tsx:121]: Something to consider with the group - did we want to display average or actual travel times for each person?

File: frontend/my-app/src/Pages/Account.tsx [hunk 4]
```diff
@@ -1,97 +1,195 @@
...
-  const [passwordConfirm, setPasswordConfirm] = useState<string>('')
+  const [newEmail, setNewEmail] = useState<string>(NEW_EMAIL)
+  const [address, setAddress] = useState<string>(location.state.location)
+  const [newAddress, setNewAddress] = useState<string>(NEW_ADDRESS)
+  const [newPassword, setNewPassword] = useState<string>(NEW_PASSWORD)
+  const [confirmPassword, setConfirmPassword] = useState<string>(NEW_CONFIRM_PASSWORD)
 
   const handleDeleteAccount = (): void => {
-    console.log('Delete Account dummy function')
+    console.log('Delete Account dummy function (STRETCH GOAL)')
```
Comment (jspaniac) [inline frontend/my-app/src/Pages/Account.tsx]: Might want to add TODOs to delete these logs later (they're also stretch goals so idk how important it is to do so lol)

File: frontend/my-app/src/Pages/CreateAccount.tsx [hunk 5]
```diff
@@ -42,32 +52,105 @@ const CreateAccount: React.FC = () => {
...
+                />
+                <TextField
+                  margin="normal"
+                  required
+                  fullWidth
+                  name="address"
+                  label="address"
+                  type="address"
+                  id="address"
+                  autoComplete="address"
```
Comment (jspaniac) [inline frontend/my-app/src/Pages/CreateAccount.tsx:129]: Similar to password, do we need all of these? If so disregard!

**Review by jspaniac — COMMENTED** [review]
Left some comments. One thing I noticed when checking out the branch - adding too many people and resizing the window to the most narrow leads to some funky UI stuff. Would be ok making an issue for that and dealing with it later though

File: frontend/my-app/src/Pages/CreateAccount.tsx [hunk 6]
```diff
@@ -42,32 +52,105 @@ const CreateAccount: React.FC = () => {
...
+                  sx={{ input: { color: 'white' } }}
+                />
+              </ThemeProvider>
+            <Button
+              onClick={handleCreate}
+              fullWidth
+              variant="contained"
+              sx={{ mt: 3, mb: 2, bgcolor: 'secondary.main' }}
+            >
+              Create Account
```
Comment (zshafe) [inline frontend/my-app/src/Pages/CreateAccount.tsx:142]: I think we'll need to add input verification on these inputs but that can be done later

**Review by zshafe — APPROVED** [review]

**Pull request merged**